lib/zuora-core/src/main/scala/com/gu/zuora/Zuora.scala

```diff
@@ -51,17 +51,21 @@ object Zuora {
   }
 
   def subscriptionGetResponse(config: ZuoraConfig, accessToken: AccessToken, backend: SttpBackend[Identity, Any])(subscriptionName: SubscriptionName): ZuoraApiResponse[Subscription] = {
-    basicRequest.get(uri"${config.baseUrl}/subscriptions/${subscriptionName.value}")
+    val response = basicRequest.get(uri"${config.baseUrl}/subscriptions/${subscriptionName.value}")
       .header("Authorization", s"Bearer ${accessToken.access_token}")
       .response(asJson[Subscription])
       .mapResponse(_.left.map(e => ZuoraApiFailure(e.getMessage)))
       .send(backend)
-      .body
+
+    logger.info(s"subscriptionGetResponse for sub ${subscriptionName.value} is ${response.body.toString}")
```

Comment:
You might get more readable output with something like 
```suggestion
    import io.circe.syntax._ //stick this at the top of the file
    logger.info(s"subscriptionGetResponse for sub ${subscriptionName.value} is ${response.body.asJson}")
```